UnityProject/Assets/Scripts/Objects/PushPull.cs

```diff
@@ -96,12 +99,31 @@ public class PushPull : VisibleBehaviour {
 				PlayerManager.LocalPlayerScript.soundNetworkActions.RpcPlayNetworkSound("Rustle0" + Random.Range(1, 4), pullable.transform.position);
 
 				PulledObject = pullable;
+
 				//Kill its impulses if we grabbed it
 				PulledObject.Stop();
+
+				// Update the UI
+				UpdatePullingUI(this);
 			}
 		}
 	}
 
+	[Server]
+	private void UpdatePullingUI(PushPull pull)
+	{
+		ConnectedPlayer player = PlayerList.Instance.Get(pull.gameObject);
+
+		if (player != ConnectedPlayer.Invalid)
+			TargetUpdatePullingUI(player.Connection, pull.IsPullingSomething);
+	}
+
+	[TargetRpc]
```

Comment:
Please refrain from using RPC further on as it's proven to be unreliable; 
Plus there's already an InformPullMessage and its `Process()` method that gets executed on client after he receives a message that `Subject` is now pulled by `PulledBy`, so it would be correct to update UI there, in one place, according to received info:
https://github.com/unitystation/unitystation/blob/69fcf3d253771417748adde720a4ce9865e32272/UnityProject/Assets/Scripts/Messages/Server/InformPullMessage.cs#L28

Reply:
Yeah, sorry about that one! We should probably put the information about Rpcs being unreliable somewhere in the contributing guidelines as it is not something I could find anywhere else on the internet.